### ISSUE TITLE
Find (if supported) address of KaKu type APA3-1500R receiving unit

### ISSUE THREAD

**Issue by ChielChiel** [robertklep/homebridge-klikaanklikuit#35]
Hi, I've been struggling with this for a while now: I'm unable to locate the address of my KaKu devices.
I use [this remote: AYCT-102](https://www.conrad.nl/p/klikaanklikuit-ayct-102-afstandsbediening-70056-1509572) with these [outlets: APA3-1500R].

But somehow I'm unable to activate the outlets even when trying to use [`Kaku-rpi`](https://github.com/robertklep/node-kaku-rpi) on my RaspberryPi 3b with [this transmitter](https://www.amazon.co.uk/dp/B07B9KV8D9/). I connected the datapin to pin 11 (GPIO pin 17) and I have tried all of these commands:
``` shell
$ sudo kaku 11 [A-D] [1-4] 1
```
But to no avail.

I have confirmed that the transmitter is working with the [`rpi-rf`](https://github.com/milaq/rpi-rf) module

Hope somebody can point me in the right direction!

Btw, I would suggest creating a **wiki** with all the combined knowledge about supported devices and common issues etc. It's easier than to search through the issues.

**Comment by robertklep** [comment]
What I usually do is just assign a new code to the outlet: put it in learning mode, and send a new code using `kaku`.

> Btw, I would suggest creating a wiki with all the combined knowledge about supported devices and common issues etc. It's easier than to search through the issues.

I agree, but since I don't use KaKu myself anymore this repository has led a dormant state.

**Comment by ChielChiel** [comment]
Strange, whenever i put it in learning mode (pull the outlet out of the wall socket and put it, after 10 seconds, back in) and send a command like the one above, nothing happens. 
Any advice?

**Comment by robertklep** [comment]
Try using a numerical address:
```
sudo kaku 11 1234 1 1
```

**Comment by ChielChiel** [comment]
Does it matter what kind of numerical address I send matter? For example could it be `4321` as well?

**Comment by robertklep** [comment]
Yes, that's fine too :) I don't know what the maximum allowed number is though, I think it's 16384

**Comment by ChielChiel** [comment]
Weird, when sending:
`sudo kaku 8 1101 1 1`
Right after i plugged in the outlet, nothing happens: `sudo kaku 8 1101 1 0` won't switch it off

**Comment by robertklep** [comment]
Did you change the pin from 11 to 8? Also, does the device put itself in learning mode automatically when you power it up? It needs to be put in that mode before you send the code, so it can learn it.

**Comment by ChielChiel** [comment]
Yep, i changed the pin just to check if the result would be different. It is the physical pin `8`. According to the manual of this device, it is automatically put in learning-mode right after you plug it into the wall and 10 secs after it was last pulled out. So I handled accordingly...

**Comment by robertklep** [comment]
Hmm in that case, I have no idea why it's not working. Because I don't own any KaKu devices anymore, I can't check myself either :(
